__tests__/app.test.js

```diff
@@ -341,6 +341,23 @@ describe("app", () => {
       });
     });
   });
+  describe("/api/users", () => {
+    describe("GET requests /api/users", () => {
+      test("200: responds with a users array on the response body", () => {
+        return request(app)
+          .get("/api/users")
+          .expect(200)
+          .then(({ body }) => {
+            const { users } = body;
+            users.forEach((user) => {
```

Comment:
As we're asserting in a forEach, could you please check the length of the array beforehand?